src/site/content/en/blog/advancing-framework-ecosystem-cds-2019/index.md

```diff
@@ -124,10 +122,8 @@ for comments (RFCs) and pull requests (PRs):
     ([PR](https://github.com/zeit/next.js/pull/8069)).
 
 <figure class="w-figure">
-  <img src="barnebys.png" class="w-screenshot-filled" alt="Homepage of Barnebys.com">
-  <figcaption class="w-figcaption">
-    <a href="https://www.barnebys.com/">Barnebys.com</a>, a large search engine for antiques and collectibles, saw a 23% reduction in total JavaScript after enabling granular chunking
-  </figcaption>
+  {% Img src="image/admin/5QA6KNYCwQ4aLdowLGNS.png", alt="Homepage of Barnebys.com", width="800", height="543", class="w-screenshot-filled" %}
```

Comment:
I think the original author may have wanted to use `w-screenshot w-screenshot--filled` here but didn't